tools/coqdep.ml

```diff
@@ -8,84 +8,10 @@
-  eprintf "  -noglob | -no-glob : \n";
-  eprintf "  -noinit : currently no effect\n";
-  eprintf "  -f file : read -I, -Q, -R and filenames from _CoqProject-formatted FILE.\n";
-  eprintf "  -I dir : add (non recursively) dir to ocaml path\n";
-  eprintf "  -R dir logname : add and import dir recursively to coq load path under logical name logname\n";
-  eprintf "  -Q dir logname : add (recursively) and open (non recursively) dir to coq load path under logical name logname\n";
-  eprintf "  -vos : also output dependencies about .vos files\n";
-  eprintf "  -exclude-dir dir : skip subdirectories named 'dir' during -R/-Q search\n";
-  eprintf "  -coqlib dir : set the coq standard library directory\n";
-  eprintf "  -dyndep (opt|byte|both|no|var) : set how dependencies over ML modules are printed\n";
-  exit 1
-
-let split_period = Str.split (Str.regexp (Str.quote "."))
-
-let add_q_include path l = add_rec_dir_no_import add_known path (split_period l)
-let add_r_include path l = add_rec_dir_import add_known path (split_period l)
-
-let treat_coqproject f =
-  let open CoqProject_file in
-  let iter_sourced f = List.iter (fun {thing} -> f thing) in
-  let warning_fn x = coqdep_warning "%s" x in
-  let project = read_project_file ~warning_fn f in
-  iter_sourced (fun { path } -> add_caml_dir path) project.ml_includes;
-  iter_sourced (fun ({ path }, l) -> add_q_include path l) project.q_includes;
-  iter_sourced (fun ({ path }, l) -> add_r_include path l) project.r_includes;
-  iter_sourced (fun f -> treat_file None f) (all_files project)
-
-let rec parse = function
-  | "-boot" :: ll -> option_boot := true; parse ll
-  | "-sort" :: ll -> option_sort := true; parse ll
-  | "-vos" :: ll -> write_vos := true; parse ll
-  | ("-noglob" | "-no-glob") :: ll -> option_noglob := true; parse ll
-  | "-noinit" :: ll -> (* do nothing *) parse ll
-  | "-f" :: f :: ll -> treat_coqproject f; parse ll
-  | "-I" :: r :: ll -> add_caml_dir r; parse ll
-  | "-I" :: [] -> usage ()
-  | "-R" :: r :: ln :: ll -> add_r_include r ln; parse ll
-  | "-Q" :: r :: ln :: ll -> add_q_include r ln; parse ll
-  | "-R" :: ([] | [_]) -> usage ()
-  | "-exclude-dir" :: r :: ll -> System.exclude_directory r; parse ll
-  | "-exclude-dir" :: [] -> usage ()
-  | "-coqlib" :: r :: ll -> Envars.set_user_coqlib r; parse ll
-  | "-coqlib" :: [] -> usage ()
-  | "-dyndep" :: "no" :: ll -> option_dynlink := No; parse ll
-  | "-dyndep" :: "opt" :: ll -> option_dynlink := Opt; parse ll
-  | "-dyndep" :: "byte" :: ll -> option_dynlink := Byte; parse ll
-  | "-dyndep" :: "both" :: ll -> option_dynlink := Both; parse ll
-  | "-dyndep" :: "var" :: ll -> option_dynlink := Variable; parse ll
-  | ("-h"|"--help"|"-help") :: _ -> usage ()
-  | opt :: ll when String.length opt > 0 && opt.[0] = '-' ->
-     coqdep_warning "unknown option %s" opt; parse ll
-  | f :: ll -> treat_file None f; parse ll
-  | [] -> ()
+let (//) = Filename.concat
```

Comment:
Why not `(/)`? I don't think we use integer division in coqdep.

Reply:
I kept `//` as it was used already in the code with this particular format.